.tool-versions

```diff
@@ -1,6 +1,6 @@
 # These are used for the dev environment.
 # This should match the versions used in the built product.
 nodejs 14.18.3
-elixir ref:v1.13.4
-erlang 25.0-rc3
+elixir 1.13.4-otp-24
+erlang 24.3.4
```

Comment:
@princemaple Thanks! I was using OTP because it has JIT support for ARM64 (for Apple devices). We can hold off until it's stable.

Could you also update the versions used in the CI pipeline and the product build in these places:
- `.github/workflows/ci.yml`
- `omnibus/config/software/(elixir|erlang.rb)`

Reply:
Good thing it's right around the corner 😃